app/service/rating_service.rb

```diff
@@ -0,0 +1,22 @@
+class RatingService
+  def initialize(class_name, params)
+    @class_name = class_name
+    @params = params
+  end
+
+  def call
+    if params[:filter].present?
+      class_name.where(movie_id: params[:filter]).paginate(page: params[:page]).order("movie_id #{order}")
+    else
+      class_name.paginate(page: params[:page]).order("rating #{order}")
```

Comment:
Who in general may need the entire rating for all users and all movies?

Reply:
yeah it's useless